File: C/URI/1010.c
#include <stdio.h>
 
int main() {
 
int codigo_1, peca_1, codigo_2, peca_2;
        double valor_1, valor_2, valor_total;
        scanf("%d %d %lf", &codigo_1, &peca_1, &valor_1);
        scanf("%d %d %lf", &codigo_2, &peca_2, &valor_2);
        valor_total = 0;

        while (peca_1 > 0){
                valor_total = valor_total + valor_1;
                peca_1 = peca_1 - 1;
        }

        while (peca_2 > 0){
                valor_total = valor_total + valor_2;
                peca_2 = peca_2 - 1;
        }
        printf("VALOR A PAGAR: R$ %.2lf\n", valor_total);

    return 0;
}

File: C#/URI/1010.cs
using System; 

class URI {

    static void Main(string[] args) { 

            string[] linha1 = Console.ReadLine().Split(' ');
            int codigo = int.Parse(linha1[0]);
            int quant_pec = int.Parse(linha1[1]);
            double valor = double.Parse(linha1[2]);
            string[] linha2 = Console.ReadLine().Split(' ');
            int codigo2 = int.Parse(linha2[0]);
            int quant_pec2 = int.Parse(linha2[1]);
            double valor2 = double.Parse(linha2[2]);

            Console.WriteLine("VALOR A PAGAR: R$ {0:0.00}", _ = (quant_pec * valor) +(quant_pec2*valor2));


    }

}